modules/ztp-image-based-upgrade-upgrade.adoc

```diff
@@ -128,4 +128,61 @@ cgu-ibu-finalize        30h   Completed   All clusters are compliant with all th
 cgu-ibu-prep            31h   Completed   All clusters are compliant with all the managed policies
 cgu-ibu-upgrade         31h   Completed   All clusters are compliant with all the managed policies
 ----
---
\ No newline at end of file
+--
+
+. You can remove the {oadp-short} Operator and its configuration files after a successful upgrade.
+
+.. Change the `complianceType` to `mustnothave` for the {oadp-short} Operator namespace, Operator group, and subscription in the `common-ranGen.yaml` file.
++
+[source,yaml]
```

Comment:
Is `common-ranGen.yaml` an OpenShift API object? If so, this and the following yaml examples should include the required fields (https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#required-fields).

Reply:
[It is](https://github.com/openshift-kni/cnf-features-deploy/blob/master/ztp/policygenerator-kustomize-plugin/testPolicyGenTemplate/common-ranGen.yaml) and it's a file that is provided in the `ztp-site-generate` container by default so I'd think that the context is clear. However, we don't want to include the full content of the file, only the relevant bits, so I'm putting `[...]` at the beginning and end